tools/Environments/DevHome.Environments/ViewModels/LandingPageViewModel.cs

```diff
@@ -157,22 +250,61 @@ public void SearchHandler(string query)
         };
     }
 
+    /// <summary>
+    /// Updates the view model to filter the compute systems according to the provider.
+    /// </summary>
+    [RelayCommand]
+    public void ProviderHandler()
+    {
+        var currentProvider = Providers[SelectedProviderIndex];
+        ComputeSystemsView.Filter = system =>
+        {
+            if (currentProvider == _stringResource.GetLocalized("AllProviders"))
+            {
+                return true;
+            }
+
+            if (system is ComputeSystemViewModel computeSystemViewModel)
+            {
+                var type = computeSystemViewModel.Type;
+                return type.Contains(currentProvider, StringComparison.OrdinalIgnoreCase);

```

Comment:
Seems a bit weird to do Contains instead of equals. Though I know in the grand scheme of things we don't expect one provider's name to be a subset of another providers name. Using equals ensures we're only showing ones which the provider name only

Reply:
The idea was to not care if they have a longer provider name. But equals would be better.
Changed.